frontend/src/viewer/logic/score/league/czech.ts

```diff
@@ -0,0 +1,62 @@
+import { Measure } from '../measure';
+import { Score, scoreCircuits, scoreOpenDistance, scoreTriangles } from '../scorer';
+
+abstract class CZXCBase {
+  score(measure: Measure): Score[] {
+    return [
+      ...scoreOpenDistance(measure, 3, this.openDistanceMultiplier),
+      ...scoreCircuits(measure, null, this.scoreTriangles),
+    ];
+  }
+  protected scoreTriangles = (measure: Measure, distances: number[], indexes: number[]): Score[] =>
+    scoreTriangles(this.faiTriangleMultiplier(), this.flatTriangleMultiplier(), 0.05, false, true, measure, distances, indexes);
+
+  protected abstract faiTriangleMultiplier(): number;
+  protected abstract flatTriangleMultiplier(): number;
+  protected abstract openDistanceMultiplier(): number;
+
+}
+
+
+export class CzechLocal extends CZXCBase {
+  name = 'Czech (ČPP local)';
+
+  faiTriangleMultiplier(): number {
```

Comment:
Could you please add the missing `protected` everywhere ?

Reply:
will do.